scripts/serie_update.py

```diff
@@ -0,0 +1,590 @@
+#!/usr/bin/python
+
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2019 STMicroelectronics.
```

Comment:
```suggestion
# Copyright (c) 2020 STMicroelectronics.
```

Reply:
I keep 2019, as part of the code is inherited from files with 2019 copyright